src/i18n.cpp

```diff
@@ -1,6 +1,11 @@
 #include <i18n.h>
 #include <resources.h>
 
+#if defined(_MSC_VER)
+#include <cstdlib>
+#define __builtin_bswap32 _byteswap_ulong
```

Comment:
I think it would be better to add a define for both `_MSC_VER` and `__GNUC__`, and then define a bswap32 function to use in the rest of the code, instead of pretending msvc has the gcc specific builtin.